website/content/en/docs/cli/operator-sdk_run_bundle.md

```diff
@@ -10,6 +10,12 @@ Deploy an Operator in the bundle format with OLM
 The single argument to this command is a bundle image, with the full registry path specified.
 If using a docker.io image, you must specify docker.io(/&lt;namespace&gt;)?/&lt;bundle-image-name&gt;:&lt;tag&gt;.
 
+The main purpose of this command is to streamline running the bundle without having to provide an index image with the bundle already included.
+
+An index image can be specified to resolve dependencies for a bundle by specifying the `--index-image` flag.
```

Comment:
Nit:
```suggestion
The `--index-image` flag specifies an index image in which to inject the given bundle. It can be specified to resolve dependencies for a bundle. This is an optional flag which will default to `quay.io/operator-framework/opm:latest`
```

Reply:
Updated to this phrasing in commit https://github.com/operator-framework/operator-sdk/pull/5727/commits/29b7da7ade14c2eec1e627298c0a76d67cfaa7d7